scripts/shared/build_image.sh

```diff
@@ -47,17 +52,31 @@ if [[ "$cache" = true ]]; then
     done
 fi
 
+output_flag=--load
+[[ -z "${ocifile}" ]] || output_flag="--output=type=oci,dest=${ocifile}"
+
+# Default to linux/amd64 (for CI); platforms match Go OS/arch
+if command -v "${GO:-go}" >/dev/null; then
+    default_platform="$(${GO:-go} env GOOS)/$(${GO:-go} env GOARCH)"
```

Comment:
Why use default `go` if it already defaults to `go` in the Makefile? https://github.com/submariner-io/shipyard/blob/devel/Makefile.inc#L87

Reply:
So that the script can be run without necessarily relying on the `Makefile`.

Reply:
I don't believe we run these scripts outside of `make`, seems like an unnecessary complication IMHO

Reply:
I run these scripts outside of `make` ;-).